Generals/Code/GameEngine/Source/GameClient/MapUtil.cpp

```diff
@@ -81,26 +81,13 @@ static Coord3DList	m_techPositions;
 static Int m_mapDX = 0;
 static Int m_mapDY = 0;
 
-static UnsignedInt calcCRC( AsciiString dirName, AsciiString fname )
+// TheSuperHackers @refactor bobtista 31/10/2025 Remove unused variables and parameter. Function only uses fname directly.
```

Comment:
Refactor is very simple and does not need comments. There is no risk to be aware of going forward. Refactor comment can make sense when it is big and has chance of breaking something covertly.